sys-block/partclone/partclone-0.3.18.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+if [[ -z ${PV%%*9999} ]]; then
+	EGIT_REPO_URI="https://github.com/Thomas-Tsai/${PN}.git"
```

Comment:
Just a FYI
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI

Reply:
Don't know if I really understand the rational here... So would be required to replace PN variable with the expected content?
I see official ebuilds like gemato doing this
SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.tar.gz"

Reply:
Plain texted the PN. I understood now why pypi is an exception for the style guide rule.